composer.json

```diff
@@ -126,7 +127,10 @@
         ]
     },
     "autoload-dev": {
-        "files": [ "src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
+        "files": [
+            "src/Symfony/Component/VarDumper/Resources/functions/dump.php",
+            "vendor/twig/twig/src/Extension/CoreExtension.php"
```

Comment:
this makes no sense, it should be remove
you should generally consider that the code base is pretty clean and most phpstan reports should be considered first as false positives. This one is for sure.

Reply:
[Codebase is using functions](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Twig/Node/SearchAndRenderBlockNode.php#L54) defined in this file. Of course, this extension is loaded by the Twig environment, but it works only as a side-effect. This makes it explicit.

I can move it to PHPStan's own autoloader config, as Ondrej mentioned.

Reply:
please do, this is on purpose